drivers/hmis/app/graphql/types/hmis_schema/enums/hud/early_exit_reason.rb

```diff
@@ -0,0 +1,21 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+# THIS FILE IS GENERATED, DO NOT EDIT DIRECTLY
+
+module Types
+  class HmisSchema::Enums::Hud::EarlyExitReason < Types::BaseEnum
+    description '4.37.A'
```

Comment:
@eanders I'm assuming this is an old HUD list that doesnt exist in the 2022 spec? the EarlyExitReason field uses R17.A ExpelledReason list according to the spec. There are a few like this that are old. Just flagging in case it needs to be changed somehow in the warehouse.
https://github.com/greenriver/hmis-warehouse/blob/44d385fa7f1d60cb9c7693bdadb394297e0d94eb/lib/util/hud.rb#L183-L184

EarlyExitReason is R.17.A

Reply:
Yes, that is from an earlier version:
https://hudhdx.info/Resources/Vendors/5_1_2/HMISCSVSpecifications6_12.pdf
Not sure if we still need to support it, but going forward, we'll probably need a pattern to support deprecated and removed enums/columns.

Reply:
I'm inclined to have that live in the (non-generated) code, some way to translate `4.37.B` into `R17.A`. The file as generated only has the current codes from 2022 spec (some of which I went through and updated manually since the ones currently in HUD.rb are either wrong or outdated)

Reply:
I like that pattern.  If something is deprecated, we move it to the main HUD module and note when it was valid/no longer valid.